core/execution/market.go

```diff
@@ -2984,13 +2929,11 @@ func (m *Market) orderCancelReplace(
 	// the ones with the fees embedded
 	conf.Trades = trades
 
-	// pegged order might have been parked
+	// pegged order might have been parked ??? NO?
 	if m.peggedOrders.IsParked(newOrder.ID) {
 		m.peggedOrders.Unpark(newOrder.ID)
 	}
```

Comment:
Pretty sure that can't happen, as we handle parked orders earlier in the flow, by just submitting them, also we would panic a few lines before trying to remove it from book.